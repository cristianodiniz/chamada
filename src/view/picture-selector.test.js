import React from "react";
import PictureSelector from "./picture-selector";
import renderer from "react-test-renderer";
import ReactDOM from "react-dom";


describe("Picture Selector Tests Component", function() {
  it("Should render without crashing", async done => {
    const div = document.createElement("div");
    ReactDOM.render(<PictureSelector />, div);
    ReactDOM.unmountComponentAtNode(div);
    done();
  }, 10000);

  test("Should Match Snapshot", async done => {
    const component = renderer.create(<PictureSelector />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    done();
  }, 10000);
});
