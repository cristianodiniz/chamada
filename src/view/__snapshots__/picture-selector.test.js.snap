// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Picture Selector Tests Component Should Match Snapshot 1`] = `null`;
